Blazor/README.md

```diff
@@ -125,12 +123,19 @@ To use, in your project:
     ...    
 ```
   - If using the application template, simply use `appTop` and `appFooter` instead
```

Comment:
If I forget what I know and just read this sentence, it is unclear whether we are meant to do `<CDTSLayout AppTop="@topObj" ...`  or  ` GoC.WebTemplate.Blazor.Model.AppTop appTopObj = new GoC.WebTemplate.Blazor.Model.AppTop(){};` or even the inefective `GoC.WebTemplate.Blazor.Model.Top appTopObj = new GoC.WebTemplate.Blazor.Model.Top(){};`

I would do a couple of things here:
1- in the @code example, include all properties as example (you can do something like this if an empty initializer is not a good example:
```c#
GoC.WebTemplate.Blazor.Model.Top topObj = new GoC.WebTemplate.Blazor.Model.Top()
{
   ... //Top properties
};
```

2- Change this sentence to "If using the application template, simply use instances of  `AppTop` and `AppFooter` instead"

3- After the sentence, give an example of using AppTop (the variable can still be called topObj of course)
```c#
    GoC.WebTemplate.Blazor.Model.AppTop topObj = new GoC.WebTemplate.Blazor.Model.AppTop() { ... };
```

Reply:
done!